resources/ts/defs/generic.ts

```diff
@@ -1,9 +1,9 @@
 /**
  * Generic type generates all possible path combinations from object definition.
  *
- * @link https://stackoverflow.com/a/58436959
- * @typeParam T Object definition.
- * @typeParam D Recursion depth. Good to use for complex object to prevent compiler performance overload.
+ * {@link https://stackoverflow.com/a/58436959}
+ * typeParam T Object definition.
```

Comment:
Is it correct that the `@` sign is removed from `typeParam` here and in other files?

Reply:
Yes, apparently `@typeParam` isn't something that JSDoc allows.  I haven't found a way to replace this with something that passes all JSDoc checks, and I wouldn't want to delete the comment either. Therefore, I deleted the `@` sign.